tests/sentry/hybridcloud/test_app.py

```diff
@@ -42,3 +42,60 @@ def test_find_alertable_services() -> None:
     assert len(services) == 2
     assert services[0].title in [app1.name, app2.name]
     assert services[1].title in [app1.name, app2.name]
+
+
+@django_db_all(transaction=True)
+def test_get_component_context() -> None:
```

Comment:
Curious about your thoughts on function db_all tests vs all silo tests when validating RPC behavior. I vaguely remember all silo tests being slightly more exhaustive, while also being slower

Reply:
Good point about the all_silos. I'll add that.